eng/Packages.Data.props

```diff
@@ -192,6 +192,7 @@
   <ItemGroup Condition="$(MSBuildProjectName.StartsWith('Azure.Developer.MicrosoftPlaywrightTesting'))">
     <PackageReference Update="Microsoft.TestPlatform.ObjectModel" Version="17.10.0" />
     <PackageReference Update="NUnit" Version="3.13.2" />
+    <PackageReference Update="Microsoft.Extensions.Logging" Version="2.0.0" />
```

Comment:
This should be 8.x unless there's a reason to not do so.  We're in the process of bumping all of the Azure SDK dependencies to the 8.x line.

Reply:
`Microsoft.Extensions.Logging` 8.x uses `Microsoft.Bcl.AsyncInterfaces` 8.x, while in `Packages.Data.props`, `Microsoft.Bcl.AsyncInterfaces` is set to 6.x - https://github.com/Azure/azure-sdk-for-net/blob/f97d3dd8c9ba2b47675a58884171221b5d4ef927/eng/Packages.Data.props#L104

This causes a version mismatch in net6.0

```
C:\Program Files\dotnet\sdk\8.0.404\Microsoft.Common.CurrentVersion.targets(2412,5): warning MSB3277: Found conflicts b
etween different versions of "Microsoft.Bcl.AsyncInterfaces" that could not be resolved. [Dev\azure
-sdk-for-net\sdk\playwrighttesting\Azure.Developer.MicrosoftPlaywrightTesting.TestLogger\tests\Azure.Developer.Microsof
tPlaywrightTesting.TestLogger.Tests.csproj::TargetFramework=net6.0]
```

Reply:
The Async Interfaces package is in the process of being bumped to the 8.x line centrally, along with the remainder of our dependencies.   Please either wait for that to complete or bump the necessary dependencies for Playwright.    I'd rather not see us use an older set here that will need to be migrated separately.

Reply:
> This causes a version mismatch in net6.0

.NET 6 reached end-of-life and is no longer a supported platform.    Our test matrices are also in the process of being upgraded to remove 6 and include 9.